src/modules/land_detector/FixedwingLandDetector.cpp

```diff
@@ -54,7 +54,7 @@ FixedwingLandDetector::FixedwingLandDetector()
 	_paramHandle.maxVelocity = param_find("LNDFW_VEL_XY_MAX");
 	_paramHandle.maxClimbRate = param_find("LNDFW_VEL_Z_MAX");
 	_paramHandle.maxAirSpeed = param_find("LNDFW_AIRSPD_MAX");
-	_paramHandle.maxIntVelocity = param_find("LNDFW_VELI_MAX");
```

Comment:
This breaks compatibility with already configured airframes. Is renaming the param worth that?

Reply:
You or @dagar can decide that much better. But at the moment the parameter name is simply wrong.

Reply:
In this case I think the rename is justified given that both the name and description are completely wrong. Let's be sure to call it out in the release notes though.